x-pack/plugins/cases/public/methods/use_cases_toast.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiText } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { toMountPoint } from '../../../../../src/plugins/kibana_react/public';
+import { Case } from '../../common';
+import { useKibana } from '../common/lib/kibana';
+import { CasesDeepLinkId, generateCaseViewPath } from '../common/navigation';
+import { useCasesContext } from '../components/cases_context/use_cases_context';
+import { CASE_SUCCESS_SYNC_TEXT, CASE_SUCCESS_TOAST, VIEW_CASE } from './translations';
+
+const LINE_CLAMP = 3;
```

Comment:
Note: I am moving code from a place to another. I am not a fan of styled components but I don't want to introduce a refactor of code and UI change at the same time. This code comes from the existing timelines code.